app/javascript/react/components/main.js

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+import ReactDom from "react-dom"
+
+import App from "./App"
+
+ReactDom.render
+  (<App />, 
+  document.getElementById("app")
+)
```

Comment:
`/app/javascript/packs/application.js` is already set up to be the `main.js` so we can delete this file

Reply:
Deleted main.js